app.rb

```diff
@@ -13,15 +13,6 @@ def initialize
     @rentals = []
   end
 
-  def run
-    prompt_user
-  end
-
-  def exit_app
-    puts 'Thank you for using this School Library App built in Ruby!'
-    exit
-  end
-
   def list_all_people
     if @people.empty?
       puts 'There are no entries, please add people'
```

Comment:
- [ ] This violates the "Single Responsibility Principle(SRP)" which state that a class/method should have only one job. Validations should have been done in another class/method whose sole responsibility is to do validations.

- [ ] There are still rooms to refactor this class(App). Each method in this class should carry out **only** a single job. The **create_person** method should only create a person(`create_person`) without requesting inputs(choice = gets.chomp.to_i) from the user. The initial choice of options in this method below:

```ruby
  case choice
    when 1
      register_student
    when 2
      register_teacher
```

should have been done in the main.rb or menu_choice.rb file whose main purpose is to run the app, request users for their initial choice of opinions, and to quit the app.

- [ ] You'll also be required to refactor the `register_student` method. Here, you are also requesting input from the user. The line of code `student = Student.new(classroom, age, name: name, parent_permission: has_permission)` should have been placed in the create_person method whose main job is to create a person(student or teacher).
- [ ] The **register_teacher** method should also be refactored using the comments above.
- [ ] permission? the method should have been placed in the main.rb file or another class/method because the method request user's initial choice of options

Please :pray:  kindly used the comment above to refactor the rest methods in this class(App)